81E00600.4D0200.dts

```diff
@@ -0,0 +1,132 @@
+
+/dts-v1/;
+
+/ {
+        name = "";
+        compatible = "ibm,opal-in-memory-counters";
+        #address-cells = <0x1>;
+        #size-cells = <0x1>;
+        ima-nest-offset = <0x320000>;
+        ima-nest-size = <0x30000>;
+        version-id = "";
+
+        mcs {
+                compatible = "ibm,ima-counters-nest";
```

Comment:
Add "unit" and "scale" properties for the PMU nodes. They should represent the most common unit and scale values for the respective events.
For this case, the PMU node will contain scale = "1.2207e-4" and unit = ""MiB". The event nodes in this case won't have to contain the unit and scale properties.
For other cases(if any), the event node with different unit or scale values may override the values.
For cases where, even if a single event node doesn't contain the unit or scale value, the PMU must not contain the unit and scale properties and the event nodes then, will have the unit and scale values, so as to avoid ambiguity.